vars/mavenCI.groovy

```diff
@@ -61,8 +61,16 @@ def call(body) {
             }
         }
 
+        def testCmd = config.integrationTestCmd
+        if(config.runTestsInUserNamespace == true && testCmd) {
```

Comment:
1. `== true` is superfluous and can be deleted

```groovy
if ( config.runTestsInUserNamespace && testCmd) {
}
```
nit: please leave a space after keywords

Reply:
In `if (config.runTestsInUserNamespace)`,  if `runTestsInUserNamespace` has any value (non empty) then it would eveluate the if block. Hence to force boolean contract it's comparing it with `true` or `false`. 

Does that sounds right?